articlequality/feature_lists/ptwiki.py

```diff
@@ -177,7 +180,9 @@ def filter_paragraphs_without_ref_tags(segment):
     main_article_templates / max(wikitext.revision.content_chars, 1),
     (portuguese.stemmed.revision.stem_chars /
      max(wikitext.revision.content_chars, 1)),
-    log(paragraphs_without_refs_total_length + 1)
+    log(paragraphs_without_refs_total_length + 1),
+    words_to_watch_count,
+    words_to_watch_count / max(wikitext.revision.words, 1)
```

Comment:
What do you think about adding the ratio `words_to_watch_count / max(all_cite_templates, 1)`? Maybe it captures the idea that the presence of words to watch in the text is more problematic when there are few citations than when there are many.

Reply:
I think that is a great idea!